site/pages/smart-contracts/modular-account-v2/overview.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Modular Account V2
+description: An overview of the Modular Account V2 smart account.
+---
+
+# Modular Account V2 Overview
+
+Alchemy's Modular Account V2 is the most robust and feature-rich smart account on the market with top-tier enterprise grade security. Being an ERC-6900 account, MAv2 can tap into a rich ecosystem of modules comprising different authentication methods like multisig or webauthn, as well as permissions and more!
+
+With multiple developer-years of work and two comprehensive audits, MAv2 is the most advanced smart account available:
+
+- 40%+ reduced gas costs
+- Rich ecosystem of modules from teams like Circle
+- Super simple & clean developer experience
+
+## Cost
+
+Cost matters, so gas optimization was essential throughout the whole development process. As a result, MAv2 is one of the most optimized smart contracts ever written.
+
+As an example, see the gas comparison for deployment execution costs below:
+
+|                                | **Runtime gas for account creation** | **% More expensive than MAv2** |
+| ------------------------------ | ------------------------------------ | ------------------------------ |
+| **Alchemy Modular Account v2** | **97,764**                           | -                              |
+| **ZeroDev Kernel v3**          | 180,465                              | 84.6%                          |
+| **Safe**                       | 289,207                              | 195.8%                         |
```

Comment:
Could you change "Safe" to "Safe (with 4337 module)" to be more precise here?

Reply:
done here [f70cf94](https://github.com/alchemyplatform/aa-sdk/pull/1345/commits/f70cf94bd803e1a635e9ed4d90220a2c389586cb)!